src/app/main/main.component.ts

```diff
@@ -48,13 +49,15 @@ const DEBOUNCE_MS = 1000;
   templateUrl: './main.component.html',
   styleUrls: ['./main.component.css']
 })
+
 export class MainComponent implements OnInit, OnDestroy {
   readonly title = 'BigQuery Geo Viz';
   readonly StyleProps = StyleProps;
   readonly projectIDRegExp = new RegExp('^[a-z][a-z0-9\.:-]*$', 'i');
   readonly datasetIDRegExp = new RegExp('^[_a-z][a-z_0-9]*$', 'i');
   readonly tableIDRegExp = new RegExp('^[a-z][a-z_0-9]*$', 'i');
   readonly jobIDRegExp = new RegExp('[a-z0-9_-]*$', 'i');
+  readonly localStorageKey = "execution_local_storage_key";
```

Comment:
nit: ideally single quotes for consistency with the rest of the file.

Reply:
Done.